src/backend.rs

```diff
@@ -127,6 +129,89 @@ pub use self::termwiz::TermwizBackend;
 mod test;
 pub use self::test::TestBackend;
 
+static METRICS: LazyLock<Metrics> = LazyLock::new(Metrics::new);
```

Comment:
LazyLock was stabilized in Rust 1.80 which is past our current MSRV, but I'm using that for now and will replace it with lazy_static or similar until we catch up to a version that has LazyLock.